README.md

```diff
@@ -1,105 +1,65 @@
-
-See the [documentation](https://help.github.com/en/articles/metadata-syntax-for-github-actions)
-
-## Change the Code
-
-Most toolkit and CI/CD operations involve async operations so the action is run in an async function.
-
-```javascript
-import * as core from '@actions/core';
-...
-
-async function run() {
-  try { 
-      ...
-  } 
-  catch (error) {
-    core.setFailed(error.message);
-  }
-}
-
-run()
-```
-
-See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/README.md#packages) for the various packages.
-
-## Publish to a distribution branch
-
-Actions are run from GitHub repos so we will checkin the packed dist folder. 
-
-Then run [ncc](https://github.com/zeit/ncc) and push the results:
-```bash
-$ npm run package
-$ git add dist
-$ git commit -a -m "prod dependencies"
-$ git push origin releases/v1
+```yaml
+- uses: rwx-research/upload-captain-artifact@v0.0.1
+  if: always()
+  with:
+    # Required.
+    # For additional documentation on attributes see
+    # https://www.rwx.com/captain/docs/upload-artifacts-and-test-results
+    artifacts: |
+      [
+        {
+          "name": "",
+          "path": ""
+          "kind": "",
+          "parser": ""
+        }
+      ]
+
+    # Required.
+    # https://www.rwx.com/captain/docs/api-tokens
+    captain-token: ''
+
+    # This is required if you set the `name` property on your job.
+    # If you provided a name, set that same value here.
+    job-name: ''
+
+    # Optional.
+    # The behavior of the action when an artifact's file is not found.
+    #
+    #  Options:
+    #    - ignore: Output neither a warning nor an error; the action will succeed.
+    #    - warn (default): Output a warning; the action will succeed.
+    #    - error: Output an error; the action will fail.
+    if-files-not-found: ''
 ```
 
-Note: We recommend using the `--license` option for ncc, which will create a license file for all of the production node modules used in your project.
-
-Your action is now published! :rocket: 
-
-See the [versioning documentation](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md)
-
-## Validate
-
-You can now validate the action by referencing `./` in a workflow in your repo (see [test.yml](.github/workflows/test.yml))
+## Example
 
 ```yaml
-uses: ./
-with:
-  milliseconds: 1000
+- name: Upload test results to Captain
+  uses: rwx-research/upload-captain-artifact@v0.0.1
+  if: always()
+  with:
+    artifacts: |
+      [
+        {
+          "name": "RSpec",
+          "path": "tmp/rspec.json",
+          "kind": "test_results",
+          "parser": "rspec_json"
+        }
+      ]
+    captain-token: "${{ secrets.CAPTAIN_API_TOKEN }}"
 ```
 
-See the [actions tab](https://github.com/actions/typescript-action/actions) for runs of this action! :rocket:
-
-## Usage:
+You should configure the build step to run even if the test suite fails by adding `if: always()`.
```

Comment:
Should we also suggest setting `continue-on-error: true` so that we don't disrupt builds if Captain is having an outage?

Reply:
Or maybe we should handle failures on our side within the action, so that we just report them to rollbar or something and then make the step go green?

Reply:
@TAGraves it's a good idea to handle this ourselves rather than requiring people to configure it, but it might be nice for people to see that the upload failed if they happen to look at the GitHub Actions build results, which is only possible by setting `continue-on-error`. I'm inclined to recommend setting it. We have to recommend setting `if: always()` anyway, so it's not much more to add. It's also educational about best practices for configuring build steps which are basically optional :smile: